internal/constants/constants.go

```diff
@@ -287,3 +290,9 @@ const SvcAppName = "State Service"
 
 // StateAppName is the name we give our state cli executable
 const StateAppName = "State Tool"
+
+// AutoStartPath is the config key that holds the path to the auto start file
+const AutoStartPath = "auto_start_path"
+
+// InstallPath is the config key that holds the path to the installation dir
+const InstallPath = "install_path"
```

Comment:
This is being covered in Martin's PR for the new update mechanism. There'll be a new package called `installation` that provides this information.